subsys/net/lib/openthread/platform/uart.c

```diff
@@ -217,3 +226,35 @@ otError otPlatUartSend(const u8_t *aBuf, u16_t aBufLength)
 		return OT_ERROR_BUSY;
 	}
 };
+
+otError otPlatUartFlush(void)
+{
+	u32_t len;
+	const u8_t *data;
+	otError result = OT_ERROR_NONE;
+
+	len = ring_buf_get_claim(ot_uart.tx_ringbuf, (u8_t **)&data,
+				 ot_uart.tx_ringbuf->size);
+	if (len) {
+		for (size_t i = 0; i < len; i++) {
+			uart_poll_out(ot_uart.dev, data[i]);
+		}
+	} else{
+		result = OT_ERROR_INVALID_STATE;
+	}
+
+	ot_uart.tx_busy = 0;
+	atomic_set(&(ot_uart.tx_finished), 1);
+	otSysEventSignalPending();
+	return result;
+}
+
+void otPlatUartSetPanicMode(void)
```

Comment:
instead of creating OT-like function let's create Zephyr - specific function like platformUartProcess(), e.g. platformUartPanic(), it's declaration can be added to zephyr/subsys/net/lib/openthread/platform/platform-zephyr.h

Reply:
Done